py3status/modules/wwan.py

```diff
@@ -0,0 +1,564 @@
+# -*- coding: utf-8 -*-
+"""
+Display WWANs, IP addresses, signals, properties, and more.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module
+        *(default '\?color=state WW: [\?if=state_name=connected '
+        '({signal_quality_0}% at {m3gpp_operator_name}) '
+        '[{format_ipv4}[\?soft  ]{format_ipv6}]|{state_name}]'
+        '[SMS {message} [{format_message}]]')*
+    format_ipv4: display format for ipv4 network (default '[{address}]')
+    format_ipv6: display format for ipv6 network (default '[{address}]')
+    format_message: display format for SMS messages
+        (default '\?if=index<1 {contact} [\?max_length=10 {text}...]')
```

Comment:
@tobes We ought to consider if we want our indexes to be zero-based or not.

We want `1, 2, 3...` for placeholders and for color thresholds.
For conditionals, this looks clean since we want to print 1 thing.

@cyrinux, is this format correct? I recall this starts off at `1` so should this be `index<2`?

Reply:
I must remove all my sms, refind the way to do for check index<2